go.mod

```diff
@@ -11,32 +11,31 @@ require (
 	github.com/patrickmn/go-cache v2.1.1-0.20180815053127-5633e0862627+incompatible
 	github.com/pborman/uuid v1.2.1
 	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.0
-	golang.org/x/crypto v0.0.0-20220826181053-bd7e27e6170d
+	github.com/stretchr/testify v1.8.1
+	golang.org/x/crypto v0.8.0
 	gopkg.in/macaroon.v2 v2.1.0
-	gorm.io/driver/postgres v1.3.9
-	gorm.io/gorm v1.23.8
+	gorm.io/driver/postgres v1.5.2
```

Comment:
Changelog: https://github.com/go-gorm/postgres/tags

No breaking changes or other worrying things in there from what I can tell. Lots of very specific bug fixes.